src/constants.py

```diff
@@ -3,25 +3,27 @@
 
 """This module defines constants used throughout the Synapse application."""
 
-CHECK_READY_NAME = "synapse-ready"
 CHECK_ALIVE_NAME = "synapse-alive"
-CHECK_NGINX_READY_NAME = "synapse-nginx-ready"
 CHECK_MJOLNIR_READY_NAME = "synapse-mjolnir-ready"
+CHECK_NGINX_READY_NAME = "synapse-nginx-ready"
+CHECK_READY_NAME = "synapse-ready"
 COMMAND_MIGRATE_CONFIG = "migrate_config"
-PROMETHEUS_TARGET_PORT = "9000"
 SYNAPSE_CONFIG_DIR = "/data"
-SYNAPSE_CONFIG_PATH = f"{SYNAPSE_CONFIG_DIR}/homeserver.yaml"
 MJOLNIR_CONFIG_PATH = f"{SYNAPSE_CONFIG_DIR}/config/production.yaml"
-MJOLNIR_SERVICE_NAME = "mjolnir"
-MJOLNIR_USER = "mjolnir"
+MJOLNIR_HEALTH_PORT = 7777
 MJOLNIR_MANAGEMENT_ROOM = "management"
 MJOLNIR_MEMBERSHIP_ROOM = "moderators"
-MJOLNIR_HEALTH_PORT = 7777
+MJOLNIR_SERVICE_NAME = "mjolnir"
+PEER_RELATION_NAME = "synapse-peers"
+PROMETHEUS_TARGET_PORT = "9000"
+SECRET_ID = "secret-id"  # nosec
```

Comment:
We have a standard for explaining any linting disable rules, what is the reason for disabling in this case?

Reply:
Thanks for pointing that out, added!